spacy_llm/models/bedrock/model.py

```diff
@@ -0,0 +1,109 @@
+import json
+import os
+import warnings
+from enum import Enum
+from typing import Any, Dict, Iterable, List, Optional
+
+
+class Models(str, Enum):
+    # Completion models
+    TITAN_EXPRESS = "amazon.titan-text-express-v1"
+    TITAN_LITE = "amazon.titan-text-lite-v1"
+
+
+class Bedrock:
```

Comment:
For consistency we'd like all of the REST models to inherit from the `REST` class. Could you modify your `Bedrock` class to do so? You can have a look at any of the other REST-based model classes in `spacy_llm/models/rest` for inspiration (particularly the `azure` directory in particular, as it's a rather similar situation).

Reply:
Yes, sure @rmitsch, will do it.